prepare_data/src/test/groovy/org/orbisgis/bdtopo/BDTopoGISLayersTest.groovy

```diff
@@ -41,45 +44,293 @@ class BDTopoGISLayersTest {
             h2GISDatabase.load(BDTopoGISLayersTest.class.getResource("VEGET_ABSTRACT_PARAMETERS.csv"), "VEGET_ABSTRACT_PARAMETERS", true)
             h2GISDatabase.load(BDTopoGISLayersTest.class.getResource("VEGET_ABSTRACT_TYPE.csv"), "VEGET_ABSTRACT_TYPE", true)
             h2GISDatabase.load(BDTopoGISLayersTest.class.getResource("VEGET_BD_TOPO_TYPE.csv"), "VEGET_BD_TOPO_TYPE", true)
-        }
-        else{
+            */
```

Comment:
If those line are not use, they should be removed.

Reply:
We changed the approach, so they will be used